src/main/java/com/community/gateway/controller/LoginController.java

```diff
@@ -1,77 +1,79 @@
+import org.springframework.web.bind.annotation.*;
 
-import com.community.gateway.dto.OperatorDTO;
-import com.community.gateway.jwt.config.JwtUtils;
-import com.community.gateway.jwt.config.UserDetailsImpl;
-import com.community.gateway.jwt.response.JWTResponse;
-import com.community.gateway.logical.OperatorLogical;
+import javax.validation.Valid;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
 
 @CrossOrigin(origins = "http://localhost:8080")
 @RestController
 @RequestMapping("/gateway")
 public class LoginController {
 
-	@Autowired
-	private OperatorLogical loginL;
-	@Autowired
-	AuthenticationManager authenticationManager;
-	@Autowired
-	JwtUtils jwtUtils;
+    private final OperatorLogical loginL;
+    private final AuthenticationManager authenticationManager;
+    private final JwtUtils jwtUtils;
+
+    @Autowired
+    public LoginController(OperatorLogical loginL, AuthenticationManager authenticationManager, JwtUtils jwtUtils) {
+        this.loginL = loginL;
+        this.authenticationManager = authenticationManager;
+        this.jwtUtils = jwtUtils;
+    }
+
+    @PostMapping("/gw/authenticate")
+    public ResponseEntity<JWTResponse> login(@Valid @RequestBody OperatorDTO operatorRequest) {
 
-	@PostMapping("/authenticate")
-	public ResponseEntity<JWTResponse> login(@Valid @RequestBody OperatorDTO operatorRequest) {
+        OperatorDTO operator = null;
+        String jwt = null;
+        String role = null;
+        try {
+            operator = loginL.findByMobile(operatorRequest.getMobileNumber());
 
-		OperatorDTO operator = null;
-		String jwt = null;
-		String role = null ;
-		try {
-			operator = loginL.findByMobile(operatorRequest.getMobileNumber());
+            System.out.println(
+                    " Operator login :" + operatorRequest.getMobileNumber() + "  ::: " + operatorRequest.getPassword());
+            if (operatorRequest.getPassword().equals(operator.getPassword())) {
+                OperatorDTO finalOperator = operator;
+                GrantedAuthority grantedAuthority = (GrantedAuthority) () -> finalOperator.getRole().name();
+                Authentication authentication = authenticationManager
+                        .authenticate(new UsernamePasswordAuthenticationToken(
+                                operator.getMobileNumber(),
+                                operator.getPassword(), Arrays.asList(grantedAuthority)));
 
-			System.out.println(
-					" Operator login :" + operatorRequest.getMobileNumber() + "  ::: " + operatorRequest.getPassword());
-			if (operatorRequest.getPassword().equals(operator.getPassword())) {
-				Authentication authentication = authenticationManager
-						.authenticate(new UsernamePasswordAuthenticationToken(
-								operator.getOperator_Details().getOperatorName() + "-" + operator.getMobileNumber(),
-								operator.getPassword()));
+                SecurityContextHolder.getContext().setAuthentication(authentication);
```

Comment:
I am not completely aware of how you have defined object mapping. So, I have  not used the inner object operator.getOperator_Details(). Please fix hibernate issue. I just tried to fix only auth issue and got the bearer token.